docs/DeveloperGuide.md

```diff
@@ -130,7 +130,7 @@ The `Model` component,
 
 <box type="info" seamless>
 
-**Note:** An alternative (arguably, a more OOP) model is given below. It has a `Group` list in the `AddressBook`, which `Person` references. This allows `AddressBook` to only require one `Group` object per unique tag, instead of each `Person` needing their own `Tag` objects.<br>
+**Note:** An alternative (arguably, a more OOP) model is given below. It has a `Group` list in the `AddressBook`, which `Person` references. This allows `AddressBook` to only require one `Group` object per unique group, instead of each `Person` needing their own `Group` objects. It also includes hiding the `Remark` in the `Schedule` class to ensure more OOP-ness.<br>
```

Comment:
Just a clarification, what do you mean by `It also includes hiding hiding the `Remark` in the `Schedule` class to ensure more OOP-ness`?

Reply:
Because right now the person has a `remark` class; to ensure more encapsulation, we can put the remark with the schedule as it makes more sense: Each schedule has a remark; and we can use it to add more schedules i.e. each `Person` can have multiple `Schedules` that have its own `Remark`

Reply:
Okayy I got your point. Maybe can rephrase it as such: `Additionally, a remark can also be encapsulated in the Schedule class to ensure more OOP-ness by allowing each Person to have multiple Schedules with its corresponding Remark`.